src/js/disability-benefits/526EZ/helpers.jsx

```diff
@@ -517,7 +466,11 @@ const AddressViewField = ({ formData }) => {
   let zipString;
   let stateString;
   let cityString;
-  if (zipCode) zipString = `${zipCode.slice(0, 5)}-${zipCode.slice(5)}`;
+  if (zipCode) {
+    const firstFive = zipCode.slice(0, 5);
+    const lastChunk = zipCode.slice(5);
+    zipString = `${firstFive}${lastChunk ? `-${lastChunk}` : ''}`;
```

Comment:
If we only had 5 digits in the zip, it was showing up as `12345-` previously.

Reply:
:thinking: Yeah, that might clear things up a bit. I wasn't super fond of a template string nested in a template string myself.